### PR TITLE
ghostscript: update to 9.50

### PR DESCRIPTION
**Pull request by seisman**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.6 18G95
Xcode 10.3 10G8

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by Schamschula** [comment]
Have you considered revision bumping dependent ports?

For example see https://github.com/macports/macports-ports/commit/46032f4c4d85fb135a67dbf34af9284be05c23fa

**Comment by seisman** [comment]
Is there a port command updating all dependent ports? Or do i have to find and update them manually?

File: print/ghostscript/Portfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           muniversal 1.0
 
 name                ghostscript
-version             9.27
+version             9.50
 revision            1
```
Comment (mf2k) [inline print/ghostscript/Portfile]: The `revision` needs to be set back to 0 when increasing the `version`.

**Comment by Schamschula** [comment]
To get a list of affected ports

`port echo depends:ghostscript`

From there one needs to increase the port's revision by 1. In some cases one has to add a revision entry.

File: print/ghostscript/Portfile (hunk 1 above)
Comment (seisman) [reply print/ghostscript/Portfile]: Fixed.

**Comment by seisman** [comment]
@mf2k Is it required to update other dependent ports as suggested by @Schamschula?

**Comment by Schamschula** [comment]
Any ghostscript update will cause rev-updates, as the ghostscript libraries are directly linked into these ports. In the past I've pushed the ghostscript update, followed by a separate commit for the rev-bumps.

**Comment by mf2k** [comment]
> @mf2k Is it required to update other dependent ports as suggested by @Schamschula?

Yes, if the other ports need to be rebuilt, then they should all have their `revision` increased.

**Comment by seisman** [comment]
I increased the `revision` of all dependent ports, but it took too long time to rebuild. Maybe it's better to update ghostscript first, and open another several PRs to update other ports?

**Comment by cjones051073** [comment]
@seisman please rebase your branch here against master. This should fix the merge conflicts, and also remove the merge commit from the commit list, which we do not want appearing in our commit history.

**Comment by seisman** [comment]
Rebased but I kept two commits, one for updating ghostscript, and another one for update its dependent ports.

**Comment by pmetzger** [comment]
@cjones051073 @mf2k @Schamschula We really need to do the update because of security concerns. Are one or more of you comfortable with merging this (or with my merging it?)

**Comment by cjones051073** [comment]
> @cjones051073 @mf2k @Schamschula We really need to do the update because of security concerns. Are one or more of you comfortable with merging this (or with my merging it?)

I am fine with it getting merged, and dealing with any aftermath, if there is any, later on.

**Comment by pmetzger** [comment]
I'm going to merge this then. I recognize that it may create some fallout but I don't think we can put it off.

Thanks for the work doing this, @seisman! It's appreciated!

**Comment by pmetzger** [comment]
There's a slight typo in the ghostscript makefile but I'll take care of it manually. (Stray but unimportant \. )

**Pull request merged**